openstack/nova/templates/_helpers.tpl

```diff
@@ -0,0 +1,51 @@
+{{- define "rabbitmq_user" -}}
+  {{.Values.rabbitmq.users.default.user}}
+{{- end -}}
+
+{{- define "rabbitmq_password" -}}
+  {{ .Values.rabbitmq.users.default.password | default (tuple . .Values.rabbitmq.users.default.user | include "rabbitmq.password_for_user")}}
```

Comment:
Can we use the above-defined `rabbitmq_user` here? Just in case that will not stay `.Values.rabbitmq.users.default.user` forever.

Reply:
not used anymore, I remove it